masu/config.py

```diff
@@ -66,3 +66,11 @@ class Config(object):
     REPORT_PROCESSING_BATCH_SIZE = 10000
 
     AWS_DATETIME_STR_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+    # Provider Types
+    AMAZON_WEB_SERVICES = 'AWS'
```

Comment:
There are constants, and not likely to be configurable values. I don't think they belong in a config file.